packages/svelte/src/compiler/phases/3-transform/client/visitors/SvelteElement.js

```diff
@@ -83,9 +83,12 @@ export function SvelteElement(node, context) {
 	if (
 		attributes.length === 1 &&
 		attributes[0].type === 'Attribute' &&
-		attributes[0].name.toLowerCase() === 'class'
+		attributes[0].name.toLowerCase() === 'class' &&
+		attributes[0].value !== true &&
```

Comment:
I might be wrong but there should be a `is_text_attrobute` utility somewhere...can you check if that's the case and use that?

Reply:
Will change that in the morning